packages/contracts/test/exchange/core.ts

```diff
@@ -846,4 +846,61 @@ describe('Exchange', () => {
             expect(errCode).to.be.equal(ExchangeContractErrs.ERROR_ORDER_EXPIRED);
         });
     });
+
+    describe('cancelOrdersBefore', () => {
+        it('should fail to set timestamp less-or-equal to existing cancel timestamp', async () => {
+            const timestamp = new BigNumber(0);
```

Comment:
Is this `less` or `equal`? I would have to separate tests for these two conditions.